src/Uno.UWP/System/Power/Internal/BatteryChangedBroadcastReceiver.cs

```diff
@@ -0,0 +1,24 @@
+﻿#if __ANDROID__
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Android.Content;
+
+namespace Windows.System.Power
+{
+	internal class BatteryChangedBroadcastReceiver : BroadcastReceiver
+	{
+		public override void OnReceive(Context context, Intent intent)
+		{
+			if (Intent.ActionBatteryLow.Equals(
+				   intent.Action, StringComparison.InvariantCultureIgnoreCase) ||
+				Intent.ActionBatteryOkay.Equals(
+					intent.Action, StringComparison.InvariantCultureIgnoreCase))
+			{
+				Windows.System.Power.PowerManager.RaiseBatteryStatusChanged();
```

Comment:
Seems like this will raise BatteryStatusChanged a lot, if it's fired every time the charge percentage changes? Should it check if BatteryStatus is actually different?

Reply:
In fact Android does not offer continuous battery percentage monitoring, so it only reports for "low" and "okay" whatever it is. So it will actually fire very seldom. But I can add some "last value" cache (to all properties), so that the events actually trigger only when changes occur 👌👍

Reply:
Sounds good. That limitation then (and any others) should be documented as well. You can add an .md file in doc/articles/features/

Reply:
I see, will do that as well. Thanks for the tips 😉 !

Reply:
Fixed :-) . Also improved to use a `lock` statement around event `add` and `remove` to avoid race conditions